app/assets/stylesheets/questionnaire.scss

```diff
@@ -162,3 +168,36 @@ form .likert-scale {
     }
   }
 }
+
+// Fix i'tjes being on a new line
+p.option-label, div.otherwise-textfield {
+  padding-left: 35px;
+}
+[type="checkbox"] + span:not(.lever), [type="checkbox"]:not(.filled-in) + span:not(.lever), [type="radio"]:not(:checked) + span, [type="radio"]:checked + span {
+  display: inline;
+  padding-left: 0;
+  &:before, &:after {
+    top: -0.2rem;
+    left: -35px;
+  }
+}
+[type="checkbox"]:checked + span:not(.lever):before {
+  top: -0.5rem;
+  left: -40px;
+}
+
+// Make labels more clickable
+label {
+  padding-bottom: 8px;
```

Comment:
het is nu niet zo dat ze in sommige gevallen overlappen? En moeten we hier geen REM voor gebruiken? :-)

Reply:
Bwahaha. Nee die ruimte was er al vanwege margins van het element erna. De padding is puur zodat het klikbare deel groter wordt, de controls verschuiven niet van positie.

En rem eigenlijk wel, maar alle offsets en sizes rondom labels definieert materialize in pixels en dat is best wel brak eigenlijk denk ik, en misschien ook wel de reden dat die labels niet goed mee scalen met flow-text. Maar dat heeft niet veel zin om dat alleen hier aan te passen lijjkt mij.